mitodl/open-discussions
# Add coursenum to index

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of https://github.com/mitodl/course-search-utils/issues/21

#### What's this PR do?
Adds `coursenum` to the index so that we can sort on it

#### How should this be manually tested?
Run `./manage.py recreate_index --course`. You should not see any errors and the search should work as it did before
